### ISSUE TITLE
Handling batching based on varbinary fields?

### ISSUE THREAD

**Issue by idangozlan** [join-monster/join-monster#204]
Hi,

I have a case, which I can imagine that is not so popular, that I need to work with batch and making a raw query batching, my case is that I have `VARBINARY` primary key (to get the string readable value I have to select it with `HEX(Page_UUID)` and to find it by string I have to use `where Page_UUID IN (X'string')`.

When I'm trying to make a batch call, it's doing that incorrectly:
Page Schema (which batching its comments), keep in mind that `Object_UUID_FK` and `Page_UUID` are `VARBINARY`:
`Page_UUID` (unhex) is: `0006A8FB792B977401C0A8991982EFA4`
```
....
comments: {
      type: new GraphQLList(CommentSchema),
      description: 'Comments on this blog entry',
      args: connectionArgs,
      sqlBatch: {
        // which column to match up to the users
        thisKey: 'Object_UUID_FK',
        // the other column to compare to
        parentKey: 'Page_UUID',
      },
      // sqlJoin: (pageTable, commentTable) => `${pageTable}.Page_UUID = ${commentTable}.Object_UUID_FK`,
    },
....
```

SQL:
```
 SELECT
   `comments`.`Comment_UUID` AS `Comment_UUID`,
   `comments`.`Comment_UUID` AS `UUID`,
   `comments`.`CommentContent_MText` AS `text`,
   `comments`.`Object_UUID_FK` AS `Object_UUID_FK`
 FROM Comment `comments`
 WHERE `comments`.`Object_UUID_FK` IN ('0616825112143151116119216815325130239164')
```

as you can see, instead of creating batch query with 
`... IN (X'0006A8FB792B977401C0A8991982EFA4')` 
its creating it with 
`... IN ('0616825112143151116119216815325130239164')`

Is there any way to make it happen? if you have an idea, share it with me and i'll try to create PR with the feature. if it's already exist, lemme know how to do that :)

**Comment by idangozlan** [comment]
Solution found, PR soon.

**Comment by acarl005** [comment]
Merged. Published in **v2.0.9**. Thanks for the contribution!
